R/decisiontreeutils.R

```diff
@@ -361,3 +398,34 @@ outputDTResultsAlteryx <- function(results, config) {
   title(main="Pruning Plot", line=5)
   invisible(dev.off())
 }
+
+#' process for converting to results list from config and data
+#'
+#' @param config list of configuration options
+#' @param data list of datastream objects
+#' @return list of results or results
+#' @export
+process <- function(config, data) {
+  # To get run-over-run consistency, set the seed
+  set.seed(1)
+
+  config$model.name <- validName(config$model.name)
+
+  # Load the rpart library (it is included with base R but still needs to be
+  # loaded in the the process's R environment
+  # Determine if the rpart.plot package is available.
+  loadPackages("rpart, rpart.plot", "AlteryxRhelper")
+
+  AlteryxPredictive::preModelCheckDT(config, the.data)
+
+  the_lists <- AlteryxPredictive::createDTParams(config, data)
+  params <- AlteryxPredictive::paramsToDTArgs(the_lists$f, the_lists)
+  the_model <- AlteryxPredictive::doFunction(the_lists$f, params)
```

Comment:
You dont need the :: here as it is a function in the same package. In fact R CMD Check will complain.